database.py

```diff
@@ -0,0 +1,23 @@
+import sqlite3
+
+import config
+
+# connection is a static variable that is shared across all instances of DB
+# so we reuse the same connection across all users of this class
+class Database():
```

Comment:
There's more pythonic ways of doing this, but this'll do for 0.1. We can clean it up before or after release.